docs/osmosis-core/relaying/relayer-guide.md

```diff
@@ -247,37 +247,36 @@ hermes start
 ```
 
 Watch hermes' output for successfully relayed packets or any errors.
-It will try & clear any unreceived packets after startup has completed.
+It will try & clear any unreceived ("pending") packets after startup has completed.
 
 
 ## Helpful Commands
 
-Query Hermes for unreceived packets & acknowledgements (check if channels are "clear")
+Use Hermes to query for unreceived packets & acknowledgements (check if channels are "clear").
+This will query both the Osmosis and Cosmos Hub sides of the channel, printing if there is any pending packet on that path in any direction and for any packet type.
+
 ```sh
-hermes query packet unreceived-packets cosmoshub-4 transfer channel-141
-hermes query packet unreceived-acks cosmoshub-4 transfer channel-141
+hermes query packet pending --chain cosmoshub-4 --port transfer --channel channel-141
 ```
+
+You can also individually query for each direction of the channel and each packet type (sends or acknowledgements).
 ```sh
-hermes query packet unreceived-packets osmosis-1 transfer channel-0
-hermes query packet unreceived-acks osmosis-1 transfer channel-0
+hermes query packet pending-sends --chain cosmoshub-4 --port transfer --channel channel-141
+hermes query packet pending-acks --chain cosmoshub-4 --port transfer --channel channel-141
 ```
-
-Query Hermes for packet commitments:
 ```sh
-hermes query packet commitmentss cosmoshub-4 transfer channel-141
-hermes query packet commitments osmosis-1 transfer channel-0
+hermes query packet pending-sends --chain osmosis-1 --port transfer --channel channel-0
+hermes query packet pending-acks --chain osmosis-1 --port transfer --channel channel-0
 ```
 
-Clear channel (only works on hermes `v0.12.0` and higher)
+Query Hermes for packet commitments:
 ```sh
-hermes clear packets cosmoshub-4 transfer channel-141
-hermes clear packets osmosis-1 transfer channel-0
+hermes query packet commitments --chain cosmoshub-4 --port transfer --channel channel-141
+hermes query packet commitments --chain osmosis-1 --port transfer --channel channel-0
 ```
 
-Clear unreceived packets manually. *Experimental: you'll need to stop your hermes daemon for it not to get confused with account sequences.*
+Clear channel manually in both directions. Use this in case you want to clear congestion manually. Only works on hermes `v0.12.0` and higher. *You'll need to stop your Hermes daemon before using `clear packets`. This is important, otherwise the `clear packets` process will be racing with the daemeon process to access the same relayer wallet, resulting in account sequence mismatch errors.*
+
 ```sh
-hermes tx raw packet-recv osmosis-1 cosmoshub-4 transfer channel-141
-hermes tx raw packet-ack osmosis-1 cosmoshub-4 transfer channel-141
-hermes tx raw packet-recv cosmoshub-4 osmosis-1 transfer channel-0
-hermes tx raw packet-ack cosmoshub-4 osmosis-1 transfer channel-0
-```
```

Comment:
These CLI commands are very low level and not advised for end-users. Instead, `clear packets` does the job.